pyproject.toml

```diff
@@ -75,8 +74,13 @@ redis = ["redis>=5.0.0,<6.0.0"]
 
 otel = ["opentelemetry-sdk>=1.24.0,<2.0.0"]
 
+cli = [
+    "typer>=0.9,!=0.12,<1",
+    "watchfiles==0.24.0"
```

Comment:
One last thing: can we find lowest `watchfiles` version we are working with and pin it like `watchfiles>=0.X.0,<0.25.0` ?